### ISSUE TITLE
Multi folder workspace peeking

### ISSUE THREAD

**Issue by MadeManMax** [zabel-xyz/plsql-language#40]
Hi,
I've noticed that when I have workspace composed from multiple root folders I can not peek packages that are in different root then current file.
Don't know if this is a problem in extension or in VSC
Below is my workspace configuration and attached are test files.

```JSON
{
	"folders": [
		{
			"path": "."
		},
		{
			"path": "C:\\!!!WorkPlace!!!\\_ps\\2"
		},
		{
			"path": "C:\\!!!WorkPlace!!!\\_ps\\3"
		}
	]
}
[!!!WorkPlace!!!.zip](https://github.com/zabel-xyz/plsql-language/files/2021780/WorkPlace.zip)

```

**Comment by zabel-xyz** [comment]
Like you notice, by default the extension search in root folder corresponding to current file,
But you can specify where to search with settings `plsql-language.searchFolder`

**Comment by MadeManMax** [comment]
OK, thank you.
Is that setting a list or only folder path?

**Comment by zabel-xyz** [comment]
It's a folder path. 
It specifies to [glob](https://github.com/isaacs/node-glob) the param `cwd: The working directory in which to search`
So you have to specify the common directory in wich to search (avoid c.\ for performance reason, but if you have no other choice...)

**Comment by MadeManMax** [comment]
Thank you,
Will try that but I see one problem.
My workspace usually contains of:
- core code
- implementation code dedicated for customer 
- my project code for customer

All this components are in different folder trees.
If I set common path it would be like "c:\" and it will find files from different versions of application.

I will try to create links in folders, maybe that will solve my problem.

**Comment by zabel-xyz** [comment]
I see your needs...Currently there isn't any way to do that.

You need a new setting to list all folder where to search.
It's possible, but I must change a bit the way I do the search via glob.

**Comment by m-c0de** [comment]
Have the same issue with autocompletion. Do you know when the multi-project support will be added?
Awesome extension 👍

**Comment by zabel-xyz** [comment]
Fix in version 1.7.0
New setting `plsql-language.searchPaths
`Default is now to search in all workspaces, [read more](https://github.com/zabel-xyz/plsql-language/blob/master/README.md#define-search-paths)
